src/pages/proposal-details/index.tsx

```diff
@@ -0,0 +1,328 @@
+	return (
+		<DataTable
+			className={'h-full'}
+			columnDefs={columnDefs}
+			data={validatorData}
+			loading={inProgress}
+			loader={loader}
+			minWidth="1270px"
+			noData={noDataFound}
+			tableRowClassName={'my-table-row-class'}
+		/>
+	);
+};
+
+const ProposalDetailPage: FunctionComponent = observer(() => {
+	const { proposalId } = useParams<ProposalDetailParam>();
+	const { chainStore, queriesStore } = useStore();
+	const queries = queriesStore.get(chainStore.current.chainId);
+	const proposal = queries.cosmos.queryGovernance.getProposal(proposalId);
+
+	const formatLinks = (txt: string) => {
+		return txt.split(' ').map(part =>
+			URL_REGEX.test(part) ? (
+				<a className="text-primary-50" target="_blank" href={part} rel="noreferrer">
+					{part}{' '}
+				</a>
+			) : (
+				part + ' '
+			)
+		);
+	};
+
+	const descriptionFormatter = (text: string) => {
+		return text.split('\\n\\n').map((str, index) => (
+			<p className="my-1.5" key={index}>
+				{formatLinks(str)}
+			</p>
+		));
+	};
+
+	const PieChart = () => {
+		// const data = [
+		// 	{ label: 'Yes', x: 'Yes', y: voteCalculation(proposal, 'yes', true), sliceFillColor: '#34EF52' },
+		// 	{ label: 'No', x: 'No', y: voteCalculation(proposal, 'no', true), sliceFillColor: "'#EF3456" },
+		// 	{
+		// 		label: 'No with Veto',
+		// 		x: 'No V',
+		// 		y: voteCalculation(proposal, 'no_with_veto', true),
+		// 		sliceFillColor: '#CF6679',
+		// 	},
+		// 	{ label: 'Abstain', x: 'Abstain', y: voteCalculation(proposal, 'abstain', true), sliceFillColor: '#8A86FF' },
+		// ];
+
+		// return (
+		// 	<VictoryPie
+		// 		data={data}
+		// 		labelComponent={<VictoryTooltip style={{ fontSize: '30px' }} />}
+		// 		containerComponent={<VictoryContainer />}
+		// 		animate={{
+		// 			duration: 1000,
+		// 		}}
+		// 		style={{
+		// 			data: {
+		// 				fill: (d: any) => d.slice.data.sliceFillColor,
+		// 			},
+		// 		}}
+		// 	/>
+		// );
+
+		const options = {
+			animationEnabled: true,
+			theme: 'dark2',
+			backgroundColor: '#2D2755',
+			creditText: '',
+			creditHref: '',
+			data: [
+				{
+					type: 'pie',
+					indexLabel: '{label}: {y}%',
+					startAngle: -90,
+					dataPoints: [
+						{
+							y: voteCalculation(proposal, 'yes', true),
+							label: 'Yes',
+							color: '#34EF52',
+						},
+						{
+							y: voteCalculation(proposal, 'no', true),
+							label: 'No',
+							color: '#EF3456',
+						},
+						{ y: voteCalculation(proposal, 'no_with_veto', true), label: 'No with Veto', color: '#C4A46A' },
+						{ y: voteCalculation(proposal, 'abstain', true), label: 'Abstain', color: '#8A86FF' },
+					],
+				},
+			],
+		};
+
+		return (
+			<div className="chart-wrap w-full mx-auto">
+				<CanvasJSChart options={options} containerProps={{ width: '100%', height: '250px' }} />
+			</div>
+		);
+	};
+
+	if (!proposal || !queries) {
+		return <Loader />;
+	}
+
+	return (
+		<div className="flex w-full p-2.5 pt-20 md:p-0 md:pt-0">
```

Comment:
You have a lot of `divs` here, I dont think we need all of these
There's no point in nesting divs unless the divs have multiple children, all the classes in the first 3-4 divs here can go on the first one probably